src/Augurk/Managers/FeatureManager.cs

```diff
@@ -326,10 +327,9 @@ public async Task DeleteFeaturesAsync(string productName, string groupName)
         {
             using (var session = Database.DocumentStore.OpenAsyncSession())
             {
-                await session.Advanced.DocumentStore.AsyncDatabaseCommands.DeleteByIndexAsync(
-                    nameof(Features_ByTitleProductAndGroup).Replace('_', '/'),
-                    new IndexQuery() { Query = $"Product:\"{productName}\"AND Group:\"{groupName}\"" },
-                    new BulkOperationOptions() { AllowStale = true });
+                await session.Advanced.DocumentStore.Operations.SendAsync(
```

Comment:
The API changed quite a bit here. Might want to check if this is still correct.

Reply:
Code seems functionally correct. Did you test it? (Not only by removing everything, but also a scenario where part of the contents should remain.)

Reply:
I just tried deleting a feature from Augurk using the commandline. For some reason that failed with a 404 Not Found, but deleting it through the API directly works just fine. I probably passed the command line arguments wrong or something. So looks like we're good to go here.